apps/frontend/tests/unit/Sidebar.spec.ts

```diff
@@ -22,6 +24,90 @@ const wrapper: Wrapper<Vue> = shallowMount(Sidebar, {
 });
 
 describe('Sidebar tests', () => {
+  it('will properly toggle filters', () => {
```

Comment:
confirm if there are tests that notice changes in the search bar will change the sidebar and vice versa